python-threatexchange/threatexchange/cli/tests/match_cmd_test.py

```diff
@@ -31,3 +38,32 @@ def test_invalid_hash(self):
             ("-H", "video", "--", not_hash),
             f"{not_hash!r} from .* is not a valid hash for video_md5",
         )
+
+    def test_non_photo_match_with_rotations(self):
+        with tempfile.NamedTemporaryFile() as f:
+            for content_type in ["url", "text", "video"]:
+                self.assert_cli_usage_error(
+                    ("--rotations", content_type, f.name),
+                    msg_regex="--rotations flag is only available for Photo content type",
+                )
+
+    def test_photo_hash_with_rotations(self):
+        test_file = pathlib.Path(
+            __file__ + "../../../../../../pdq/data/bridge-mods/aaa-orig.jpg"
+        ).resolve()
+
+        rotated_images = PhotoContent.all_simple_rotations(test_file.read_bytes())
+
+        for rotation, image in rotated_images.items():
+            with tempfile.NamedTemporaryFile() as tmp_file:
+                tmp_file.write(image)
+
+                if rotation == RotationType.ROTATE270:
+                    rotation = RotationType.ROTATE90
+                elif rotation == RotationType.ROTATE90:
+                    rotation = RotationType.ROTATE270
+
+                self.assert_cli_output(
+                    ("--rotations", "photo", tmp_file.name),
+                    f"pdq {rotation.name} 16 (Sample Signals) INVESTIGATION_SEED",
```

Comment:
I'm not sure if the distance here will be stable (I'm actually surprised this works), but if it passes, I think this is okay for now.